provider/aws/dist/rack.json

```diff
@@ -1249,7 +1270,6 @@
       "Properties": {
         "GroupDescription": "Instances",
         "SecurityGroupIngress": [
-          { "IpProtocol": "tcp", "FromPort": "22", "ToPort": "22", "CidrIp": { "Ref": "VPCCIDR" } },
```

Comment:
which patch did this come in with? might this affect anyone that set up real ssh access?